app/javascript/pages/fullArticle.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Text } from '@patternfly/react-core';
+import colors from '../utils/colors';
+import previewImage from '../../assets/images/img1.png';
+const styles = {
+  title: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appBlue,
+    fontSize: '50px',
+    fontWeight: 'bolder',
+    marginInline: '10%'
+  },
+  img: {
+    marginInline: '25%',
+    marginTop: '2%',
+    marginBottom: '2%',
+    borderRadius: 10
+  },
+  body: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appGrey,
+    fontSize: '30px',
+    marginInline: '20%'
+  }
+};
+const FullArticle = () => {
+  const { state } = useLocation();
+  return (
+    <>
+      <Text style={styles.title}>{state.article.title}</Text>
+
+      <img src={previewImage} width={'500px'} style={styles.img} />
```

Comment:
Should it be also part of the styles?

Reply:
No, the width is a property of the <img/>
 tag, I tried it in styles, it didn't work